src/MainOpt.h

```diff
@@ -57,7 +57,7 @@ struct MainOpt {
   uri::URI CommandBrokerURI{"localhost:9092/kafka-to-nexus.command"};
 
   /// Kafka broker and topic where file writer jobs are published.
-  uri::URI JobPoolURI{""};
+  uri::URI JobPoolURI{"localhost:9092/kafka-to-nexus.jobpool"};
```

Comment:
I would really prefer that you either remove the default address here and/or make the corresponding command line argument required (it has the same effect I believe). You can make the CL argument required on line 137 of "CLIOptions.cpp".

Reply:
My previous statement was wrong. You have to make the command "required".